pyobis/dataset/dataset.py

```diff
@@ -34,8 +34,8 @@ def search(
     :param geometry: [String] Well Known Text (WKT). A WKT shape written as
         either POINT, LINESTRING, LINEARRING
         or POLYGON.
-        Example of a polygon: ((30.1 10.1, 20, 20 40, 40 40, 30.1 10.1)) would
-        be queried as http://bit.ly/1BzNwDq. Geometry, formatted as WKT or
+        Example of a polygon: ((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1)) would
+        be queried as https://api.gbif.org/v1/occurrence/search?geometry=POLYGON%28%2830.1+10.1%2C+10+20%2C+20+40%2C+40+40%2C+30.1+10.1%29%29. Geometry, formatted as WKT or
```

Comment:
@ayushanand18 the polygon in the URL was `((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1))` . I fixed the text above this and used the full quoted URL below. However, that returns a 400 with the message:

```
Polygon with clockwise exterior ring: POLYGON((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1)). Did you mean these coordinates?  (Note this is only part of the polygon or multipolygon you provided.) LINEARRING (30.1 10.1, 40 40, 20 40, 10 20, 30.1 10.1)
```

I believe that either the example was never meant to work or something changed in the server recently.